clouddq/classes/dq_configs_cache.py

```diff
@@ -210,6 +211,14 @@ def resolve_dataplex_entity_uris(  # noqa: C901
                         entity_id=entity_uri.get_db_primary_key(),
                         dataplex_entity=dataplex_entity,
                     ).to_dict()
+                    entity_uri_primary_key = entity_uri.get_db_primary_key().upper()
+                    gcs_clouddq_entity = clouddq_entity.get(entity_uri_primary_key)
+                    gcs_entity_external_table_name = f"{gcs_clouddq_entity.get('instance_name')}.{gcs_clouddq_entity.get('database_name')}.{gcs_clouddq_entity.get('table_name')}"
```

Comment:
maybe make 'get_table_name' a method in the dq_entity.DqEntity class?

Reply:
done